### ISSUE TITLE
Can't have duplicate strings in resource paths

### ISSUE THREAD

**Issue by rjernigan** [stoicflame/enunciate#384]
Hi stoicflame,

I have a path for a resource that is set up wit the following javax.ws.rs annotation:
`@Path(ApiRestPath.GET_PRODUCT)`

In ApiRestPath, the GET_PRODUCT enum looks like this:
```java
private static final String PRODUCTS_PREFIX = "products/";
private static final String PRODUCTS_BASE = PRODUCTS_PREFIX + "catalogs/{catalogId}/products";
public static final String GET_PRODUCT = PRODUCTS_BASE + "/{productId}";
````

The resource.path that's printing in my Enunciate docs output is excluding the second 'products' path.  Instead of "products/catalogs/{catalogId}/products/{productId}" I'm only getting "products/catalogs/{catalogId}/{productId}".  Changing one of the instances of "products" in the path to a unique string makes the string appear in the output.  It seems like Enunciate may be removing duplicate strings in resource paths.

**Comment by stoicflame** [comment]
That's weird. I'll take a look.

**Comment by stoicflame** [comment]
Confirmed. Wow. I wonder how long that's been there.

Fixed at 9a05ad5.

**Comment by stoicflame** [comment]
[Enunciate 2.4 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.4.0).
